l3kernel/l3box.dtx

```diff
@@ -1727,6 +1742,61 @@
 %    \end{macrocode}
 % \end{macro}
 %
+% \subsection{Vertical boxes to width}
+%
+% \begin{macro}
+%   {
+%     \vbox_set_to_wd:Nnn,  \vbox_set_to_wd:cnn,
+%     \vbox_gset_to_wd:Nnn, \vbox_gset_to_wd:cnn,
+%     \vbox_set_top_to_wd:Nnn,  \vbox_set_top_to_wd:cnn,
+%     \vbox_gset_top_to_wd:Nnn, \vbox_gset_top_to_wd:cnn,
+%   }
+%   We both set the \tn{hsize} and the box width: this covers the case where
+%   the box is empty, for example.
```

Comment:
Could mention here that this covers any case which does not start a paragraph, so empty but also nested hbox or \halign, that is arguably covered by "empty, for example."  but possibly not obvious on first reading.

Reply:
Adjusted